config/initializers/new_framework_defaults_5_1.rb

```diff
@@ -0,0 +1,14 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
Thinking of leaving this file in, since we might hypothetically want to change to the new defaults (even though I don't expect us to? Not that I know if they are good/bad defaults, but the research/debugging time it would take to know what they do is probably greater than the reward of changing them.) If we should just get rid of this file, that's okay too.

It looks like I forgot to comment out one of the two new settings also. It doesn't seem to have changed anything for our site, but I can comment it out before this gets merged.